### ISSUE TITLE
After install with composer, command not showing up

### ISSUE THREAD

**Issue by kevinc-cogapp** [previousnext/drush_cmi_tools#29]
*What I did?*
- I changed the `composer.json` file to include the repo and the require as outlined in the README
- I ran `composer require drupal/drush-cmi-tools:dev-8.x-1.x`
- I ran `drush cc drush`
- I then ran `drush cexy` 

*What did I expect?*
- Some output concerning the `cexy` command.

*What happened?*
- I got the error message `Command "cexy" is not defined.`

*What have I tried?*
- I've tried changing the drush file I'm using, specifying the specific project `vendor` file
- Changing configs
- My project is in a Vagrant VM - I've tried copying the *.inc file to the ~/.drush directory on the vagrant box to see if that would help

**Comment by kevinc-cogapp** [comment]
@lcatlett pointed me in the right direction on the drush slack channel - my issue was that I'm running Drush 9. I've asked on the PR but is the branch that makes this compatible with drush 9 likely to be merged soon?

**Comment by iamsophiesk** [comment]
In order to use the Drush 9 version, you'll need to do `composer require drupal/drush-cmi-tools:dev-8.x-2.x` 

You will also need to enable the module. I've just spent the last hour trying to figure this out.

**Comment by kevinc-cogapp** [comment]
Thank you so much! Great solution!

**Comment by AGenchev** [comment]
How did you manage to get it working ?
I did everything as directed and ended with error: `Command "cexy" is not defined.`

I run drush this way:
`php  ./vendor/drush/drush/drush.php`  from the project root (which on my site#1 is the drupal8 root, on site#2 is the composer project template root and drupal8 is in `./web`). 
Composer and drush work as expected with other commands.

I did ../composer.phar  require  drupal/drush-cmi-tools:dev-8.x-2.x and ended with folder
`./modules/contrib/drush-cmi-tools` despite drush is in `./vendor/drush/drush/` On the other site, with different layout (installed as drupal project with composer) it goes into `./web/modules/contrib/drush-cmi-tools` where `./web` is the drupal8 folder and drush lives in  `./vendor/drush/drush/drush.php`. I hand-copied it then to ./vendor/drush/contrib/ and `./vendor/drush/drush/contrib/` but I got the same error from drush - it could not find it.
